### ISSUE TITLE
Deploy Application networking step: Fix insufficient size / spacing

### ISSUE THREAD

**Issue by vyncent-t** [Azure/aks-desktop#210]
This issue is an a11y issue related to workflow **Deploy application using wizard** #197 

Description:
When following the **Deploy application using wizard** workflow in AKS desktop, navigating into a project, then clicking the "Deploy Application" button and viewing the "Deploy Application" modal. Choose "Container Image" and click Next, 

Within the modal, under the "Networking" section, notice that the **_"Target port" and "Use custom service port"_** options has been flagged with the following a11y issues:

"Touch targets do not have sufficient size or spacing."
"Touch targets with sufficient size and spacing help users who may have difficulty targeting small controls to activate the targets."

How to test:

- Navigate to projects tab
- Navigate into a project
- Click the "Deploy application" button
- View the "Deploy application" modal
- Choose "Container Image"
- View the "Basics" section and fill information
- Navigate to the "Networking" tab
- Use inspect tools and open Lighthouse
- Scan at this view

Lighthouse

<img width="847" height="871" alt="Image" src="https://github.com/user-attachments/assets/d0ff1cd0-169d-47fc-9cd3-93a182c016a4" />

View

<img width="1823" height="878" alt="Image" src="https://github.com/user-attachments/assets/9b252d5d-828f-403b-80aa-327f1cc92a5f" />

<img width="739" height="872" alt="Image" src="https://github.com/user-attachments/assets/fac6faa7-1a53-4335-bbe6-58a6bd10e22a" />

<img width="669" height="858" alt="Image" src="https://github.com/user-attachments/assets/ae7a4942-c6ce-4218-aaad-7023f2b6ad84" />
